[2025-02-08T23:05:25.423+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: dag_with_pythonv2.fireCrawl manual__2025-02-08T23:05:19.399407+00:00 [queued]>
[2025-02-08T23:05:25.444+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: dag_with_pythonv2.fireCrawl manual__2025-02-08T23:05:19.399407+00:00 [queued]>
[2025-02-08T23:05:25.445+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-02-08T23:05:25.446+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2025-02-08T23:05:25.446+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-02-08T23:05:25.474+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): fireCrawl> on 2025-02-08 23:05:19.399407+00:00
[2025-02-08T23:05:25.480+0000] {standard_task_runner.py:55} INFO - Started process 323 to run task
[2025-02-08T23:05:25.484+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_pythonv2', 'fireCrawl', 'manual__2025-02-08T23:05:19.399407+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpo7uijefy']
[2025-02-08T23:05:25.485+0000] {standard_task_runner.py:83} INFO - Job 24: Subtask fireCrawl
[2025-02-08T23:05:25.611+0000] {task_command.py:388} INFO - Running <TaskInstance: dag_with_pythonv2.fireCrawl manual__2025-02-08T23:05:19.399407+00:00 [running]> on host 1694b062ea20
[2025-02-08T23:05:25.730+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag_with_pythonv2
AIRFLOW_CTX_TASK_ID=fireCrawl
AIRFLOW_CTX_EXECUTION_DATE=2025-02-08T23:05:19.399407+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-02-08T23:05:19.399407+00:00
[2025-02-08T23:05:25.731+0000] {firecrawl.py:32} WARNING - No API key provided
[2025-02-08T23:05:25.732+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/test.py", line 21, in get_links
    app = FirecrawlApp(api_key=os.getenv('FIRECRAWL'))
  File "/home/airflow/.local/lib/python3.7/site-packages/firecrawl/firecrawl.py", line 33, in __init__
    raise ValueError('No API key provided')
ValueError: No API key provided
[2025-02-08T23:05:25.753+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=dag_with_pythonv2, task_id=fireCrawl, execution_date=20250208T230519, start_date=20250208T230525, end_date=20250208T230525
[2025-02-08T23:05:25.773+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 24 for task fireCrawl (No API key provided; 323)
[2025-02-08T23:05:25.804+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2025-02-08T23:05:25.824+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
